ios/Extensions/UIColor+Additions.h

```diff
@@ -0,0 +1,18 @@
+//
+//  UIColor+Additions.h
+//  RNJudo
+//
+//  Created by Andrei Galkin on 4/22/20.
+//  Copyright © 2020 Facebook. All rights reserved.
```

Comment:
Facebook 😮.
Consider using the Judo headline like the other Obj-C wrapper classes.

Reply:
I don't even know why it is autogenerated with Facebook!